Amplify/Categories/API/ClientBehavior/AmplifyAPICategory+GraphQLBehavior.swift

```diff
@@ -21,6 +21,13 @@ extension AmplifyAPICategory: APICategoryGraphQLBehavior {
         plugin.query(from: modelType, where: predicate, listener: listener)
     }
 
+    public func mutate<M: Model>(of model: M,
+                                 where condition: QueryPredicate? = nil,
```

Comment:
we should confirm that this is the case, before finalizing on adding the method overload, that swift does not support omitting the parameter where there is a default parameter.